src/openforms/contrib/brk/tests/test_validators.py

```diff
@@ -0,0 +1,180 @@
+from unittest.mock import patch
+
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.utils.translation import gettext as _
+
+import requests_mock
+from privates.test import temp_private_root
+
+from openforms.authentication.constants import AuthAttribute
+from openforms.contrib.brk.models import BRKConfig
+from openforms.submissions.tests.factories import SubmissionFactory
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from ..validators import BRKZakelijkGerechtigdeValidator
+from .base import TEST_FILES, BRKTestMixin
+
+
+@temp_private_root()
+class BRKValidatorTestCase(OFVCRMixin, BRKTestMixin, TestCase):
+    VCR_TEST_FILES = TEST_FILES
+
+    def test_brk_validator_serializer(self):
+        serializer = BRKZakelijkGerechtigdeValidator.value_serializer(
+            data={"value": {"postcode": "1234 aB", "house_number": "1"}}
+        )
+        serializer.is_valid(raise_exception=True)
+        self.assertEqual(
+            serializer.data, {"value": {"postcode": "1234AB", "house_number": "1"}}
+        )
```

Comment:
See the other remark, I'd rather see a new test module created: `test_integration.py` where you set up a submission and call the validation endpoint with this validator. The implementation details on the serializer are irrelevant, what you care about is that only normalized data is sent to the BRK API & produces useful results/outcomes.